### PR TITLE
Refactor/105 nginx keda

### PR DESCRIPTION
**Pull request by Zvikan**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by kcoleman731 — APPROVED** [review]

**Review by askulkarni2 — APPROVED** [review]
Any update to `docs/` required?

File: kubernetes-addons/ingress-nginx/locals.tf [hunk 1]
```diff
@@ -2,7 +2,7 @@
 locals {
   default_helm_values = [templatefile("${path.module}/values.yaml", {})]
```
Comment (vara-bonthu) [inline kubernetes-addons/ingress-nginx/locals.tf:3]: Should this be default_helm_provider_config_values for naming consistency?

**Review by vara-bonthu — COMMENTED** [review]
LGTM except one comment on default values variable. Let's agree with one name and merge it

**Comment by Zvikan** [comment]
> Any update to `docs/` required?

we are not changing the top level variables, so there's no change required for docs.
For example: for [keda](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/main/docs/add-ons/keda.md) it will still require `keda_helm_chart` because of https://github.com/Zvikan/aws-eks-accelerator-for-terraform/blob/refactor/105-nginx-keda/kubernetes-addons.tf#L117

**Review by kcoleman731 — APPROVED** [review]

**Pull request merged**